Documentation/Extensions/SceneTransitionService/SceneTransitionServiceOverview.md

```diff
@@ -0,0 +1,187 @@
+# Scene Transition Service
```

Comment:
Consider also having an example set of scenes and "uber" scene that ties all of this together.

As a reader/someone who has zero experience using this, it's still hard for me to understand the whole picture here and how it's supposed to be used. The docs below help tremendously once you have a better understanding of the area (i.e. you know what you don't know).

Also, having some discussion about how this relates to additive scene loading (it doesn't) and the cases in which you might want to do this vs. the other would help to give a little more context.

The asks in this comment aren't something that I think need to be addressed in this PR, but should be in a subsequent on to improve the initial approachability of this feature.